translation/en/RedirectedResources/assets/abdata/communication/info_00/communication_06/translation.txt

```diff
@@ -2,7 +2,7 @@
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
 「きゃっ！　こ、ここは女性専用です、よく表示を見てから入るようにしてください」=“Kyah--! T-This one is for ladies only, please look at the sign before entering.”
-「！？　女子更衣室に入ってくるなんて……間違えたのでしたら、すぐに退出してください」=“Wha!? I cannot believe you'd walk straight into the girls' locker room... If you just got the wrong door, please leave right away.”
+「！？　女子更衣室に入ってくるなんて……間違えたのでしたら、すぐに退出してください」=“?! I cannot believe you'd walk straight into the girls' locker room... If you just got the wrong door, please leave right away.”
```

Comment:
I think this worked better before. Haven't really seen `?!` used alone in English.

Reply:
```suggestion
「！？　女子更衣室に入ってくるなんて……間違えたのでしたら、すぐに退出してください」=“Wha?! I cannot believe you'd walk straight into the girls' locker room... If you just got the wrong door, please leave right away.”
```